client/src/pages/Cart.jsx

```diff
@@ -6,16 +6,25 @@ import { usePayment } from '../features/cart/usePayment';
 import Spinner from '../ui/Spinner';
 import { formatCurrency, formatDateTimeDetailed } from '../utils/helpers';
 import { Link } from 'react-router-dom';
+import { Context } from '../main';
+import { useContext } from 'react';
 
 export default function Cart() {
+  const { isDarkMode, setIsDarkMode } = useContext(Context);
+
   const { cart, isLoading } = useCart();
   const { deleteFromCart, isDeleting } = useDeleteCartItem();
   const { clearCart, isClearing } = useClearCart();
   const { initiatePayment } = usePayment();
 
   return (
     <PageLayout>
-      <h2 className="text-3xl font-bold mt-4">Your Cart</h2>
+     <div className="w-[80%] mx-auto"
+      style={{
```

Comment:
Avoid using style on all page components to ensure proper maintainability. Make use of tailwind classes instead.